packages/fluentui/react-northstar/src/components/Form/FormLabel.tsx

```diff
@@ -2,18 +2,27 @@ import { compose, ComponentWithAs, ShorthandConfig } from '@fluentui/react-bindi
 import { commonPropTypes } from '../../utils';
 import Box, { BoxProps } from '../Box/Box';
 
-interface FormLabelOwnProps {}
+interface FormLabelOwnProps {
+  inline?: boolean;
+  required?: boolean;
+  type?: string;
```

Comment:
Let's avoid to have this prop as there is no sense in it. As an option we can remove it